lib/rspec/core/configuration.rb

```diff
@@ -778,7 +780,7 @@ def full_backtrace=(true_or_false)
         @backtrace_formatter.full_backtrace = true_or_false
       end
 
-      # Returns the configuration option for color, but should not
+      # Deprecated: returns the configuration option for color, but should not
```

Comment:
YARD has built in support for a `@deprecated` tag. Can you use that instead?  Example:

https://github.com/rspec/rspec-core/blob/6a4376e276e52b89d35aa5e16b74758e9f64b991/lib/rspec/core/example.rb#L108

http://rspec.info/documentation/3.5/rspec-core/RSpec/Core/Example.html#rerun_argument-instance_method